src/main.rs

```diff
@@ -204,11 +210,34 @@ fn install(args: InstallOpts) -> Result<()> {
             extra_crates = Some(crates);
         };
     } else {
-        exports.extend(install_gcc_targets(&targets, &host_triple)?);
+        for target in &targets {
+            let gcc = Gcc::new(target, &host_triple);
+            to_install.push(Box::new(gcc));
+        }
     }
 
     if let Some(ref extra_crates) = &extra_crates {
-        install_extra_crates(extra_crates)?;
+        for krate in extra_crates {
```

Comment:
Nitpick: typo `krate` instead of `crate`.

Reply:
I think this is on purpose because `crate` is a reserved keyword :). FYI it could be done with [raw-identifiers](https://doc.rust-lang.org/beta/rust-by-example/compatibility/raw_identifiers.html), i.e `for r#crate in extra_crates`.

Reply:
Oh, you are right, sorry 😅

Reply:
Oh, I didn't know about raw-identifiers, thanks for pointing it out.  Sadly, seems like `crate` can not be used as one:
```
error: `crate` cannot be a raw identifier
   --> src\main.rs:220:13
    |
220 |         for r#crate in extra_crates {
    |             ^^^^^^^

error: could not compile `espup` due to previous error
```

[It looks like there is a list of keywords that are not supported in r#](https://internals.rust-lang.org/t/raw-identifiers-dont-work-for-all-identifiers/9094/2)

Reply:
Interesting, I didn't know there were limitations, thanks!

Reply:
That's hilarious, I made a "stupid" review, and still all of us learned something new from it 🤣

Reply:
That's the reason why there are no "stupid" reviews! Thanks for your review!